src/javascript/JContent/ContentRoute/ContentBreadcrumb/ContentPath/ContentPath.container.jsx

```diff
@@ -47,20 +49,34 @@ const ContentPathContainer = () => {
     }));
 
     const {data, error} = useQuery(GetContentPath, {
-        variables: {path, language}
+        variables: {
+            path: externalPath || path,
+            language
+        }
     });
 
     const handleNavigation = path => {
-        dispatch(cmGoto({mode, path}));
+        dispatch(cmGoto({
+            mode: mode || JContentConstants.mode.CONTENT_FOLDERS,
+            path
+        }));
     };
 
     if (error) {
         console.log(error);
     }
 
     const node = data?.jcr?.node || {};
-    const items = useMemo(() => getItems(node), [node]);
+    const items = useMemo(() => getItems(externalPath, node), [node, externalPath]);
```

Comment:
You should use the node returned by useQuery here... I don't understand the need for passing externalPath.